projects/ngx-feature-flag/src/lib/service/enable-feature.service.ts

```diff
@@ -0,0 +1,92 @@
+import { HttpClient, HttpErrorResponse, HttpHeaders } from '@angular/common/http';
+import { Inject, Injectable, OnDestroy } from '@angular/core';
+import { cloneDeep } from 'lodash';
+import { Logger } from 'ngx-base';
+import { AuthenticationService, Profile, User, UserService } from 'ngx-login-client';
+import {
+  ConnectableObservable,
+  Observable,
+  Subscription,
+  throwError as observableThrowError
+} from 'rxjs';
+import { catchError, map, publish } from 'rxjs/operators';
+import { FABRIC8_FEATURE_TOGGLES_API_URL } from './feature-toggles.service';
+
+interface ExtUserResponse {
+  data: ExtUser;
+}
+
+export class ExtUser extends User {
+  attributes: ExtProfile;
+}
+
+export class ExtProfile extends Profile {
+  contextInformation: any;
+  registrationCompleted: boolean;
+  featureLevel: string;
+}
+
+@Injectable()
+export class EnableFeatureService implements OnDestroy {
+  private headers = new HttpHeaders({ 'Content-Type': 'application/json' });
+  protected subscriptions: Subscription[] = [];
+  private usersUrl: string;
+
+  constructor(
+    protected auth: AuthenticationService,
+    protected http: HttpClient,
+    protected logger: Logger,
+    protected userService: UserService,
+    @Inject(FABRIC8_FEATURE_TOGGLES_API_URL) apiUrl: string
+  ) {
+    if (this.auth.getToken() != undefined) {
+      this.headers = this.headers.set('Authorization', 'Bearer ' + this.auth.getToken());
```

Comment:
We don't need this as the auth interceptor should take care of adding the `Authorization` header